### DISCUSSION TITLE
Zombie notification

### DISCUSSION THREAD

**Discussion by marocchino** [community/community#49466] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have an zombie notification.(it is not shown in unread list but display as 1 always)

<img width="1508" alt="Screenshot 2023-03-08 at 10 38 43" src="https://user-images.githubusercontent.com/128431/223597177-29d5a207-2737-4f29-af4b-d8072f05736e.png">

<img width="268" alt="Screenshot 2023-03-08 at 10 38 47" src="https://user-images.githubusercontent.com/128431/223597180-e16e18e9-3900-46fb-8d4a-cdc683f09194.png">

Because of this bug, I can't check if there is actually has an new notification before clicking on the Notifications section.
I tried ignoring that repository, but it didn't change the state.
I tried [this one](https://github.com/orgs/community/discussions/43032#discussioncomment-4564181) but it is not worked for me.
Is there anything else I can try?

**Comment by deathaxe** [comment]
I also faced the zombi notification issue caused by a active notification belonging to a removed organization's repository, thus manual move to inbox and dismiss suggenstions failed.

I was finally able to resolve the issue with following steps:

1. Install the [GitHub CLI](https://cli.github.com/).
2. Authenticate from GitHub CLI.
3. Use `gh api notifications` to get a JSON containing all your notifications.
4. Note the notification (thread) ID (`<id>`) of the bad repository.
5. Execute `gh api --method PATCH notifications/threads/<id>` to read all the notifications of the given thread ID.

Note: gh might ask to upgrade your authentication to be able to modify notifications.
